cpp/daal/src/algorithms/svm/svm_predict_impl.i

```diff
@@ -204,9 +204,8 @@ struct SVMPredictImpl<defaultDense, algorithmFPType, cpu> : public Kernel
         const size_t nVectors = xTable->getNumberOfRows();
         const size_t nSV      = svTable->getNumberOfRows();
 
-        size_t nRowsPerBlock = 0;
-        DAAL_SAFE_CPU_CALL((nRowsPerBlock = 128), (nRowsPerBlock = nVectors));
-        const size_t nBlocks = nVectors / nRowsPerBlock + !!(nVectors % nRowsPerBlock);
+        const size_t nRowsPerBlock = 128;
```

Comment:
Should it be fixed? Are any empiricisms that can help to get this constant based on HW properties & data size?

Reply:
Yes, we can. In the future it is planned